validators/smoke/genesis-chain/test.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 # Simply query the 5th block and check that it's correct
-reply=`curl -s -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["0x5", false],"id":0}' "$HIVE_CLIENT_IP:8545"`
+reply=`curl -s -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["0x5", false],"id":1}' "$HIVE_CLIENT_IP:8545"`
```

Comment:
I thought it'd make sense to change this test to use id=1 as that's what we use in the [consensus tests](https://github.com/karalabe/hive/blob/master/simulators/ethereum/consensus/hivemodel.py#L19)